client/src/pages/Home/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Unused?

Reply:
Forgot to roll back this change when I went back to an older approach